README.md

```diff
@@ -1,2 +1,3 @@
 # AI-Budgeting-Assistant
 A budget analyzer using AI to make budgeting reocmmendations
+Christiny testing pushing to forked repo
```

Comment:
please remove line, you can edit "reocmmendations" spelling and use that to raise a PR